controller/partyController.js

```diff
@@ -0,0 +1,29 @@
+import db from "../db/db"
```

Comment:
Nice work @fantastic-genius , However,  I can't find where you are persisting the party you created into the  data structure database. Please do that, thanks.

Reply:
I don't understand the data structure persistence. I will appreciate if you could help out with resources on it. However, in the challenge 2 requirement as contained in what was sent across to us,  we are to use data structures without persistent

Reply:
I can see that you have done that on this P.R #46